### ISSUE TITLE
Reprdouce results

### ISSUE THREAD

**Issue by EyalMichaeli** [Zhicaiwww/Diff-Mix#3]
Hello, 

I'd like to reproduce the results.
I just want to make sure what are the correct params: how can I know what are the exact params to be used for each scenrio? are the default ones in scripts/classification.sh are the ones to use?

**Comment by EyalMichaeli** [comment]
also, with how much seeds are the results I the paper?

**Comment by Zhicaiwww** [comment]
> scripts
To reproduce the results, the most important params are learning rate, synthetic_prob and $gamma$. Basically, you can reproduce using the script run.sh.

1.  For conventional classification, set a smaller synthetic_prob (e.g., 0.1 across all datasets). For imbalance and few-shot classification, synthetic_prob should be greater (e.g., 0.3 for 5-shot and 0.3 for IF=0.01 imbalance classification).
2.  Strength should be set larger, and 0.7 is good across various scenarios.
3. $gamma$ should be set smaller than 1, and 0.5 is a safe choice (or 0.8, we alternatively choose between them). 
4. The function get_learning_rate() in scripts/classification.sh automatically handles the learning rate setup for all datasets (under conventional and few-shot classification). Please check spelling and errors.
5. We use seeds 2020, 2021, 2022 in all experiments.

**Comment by NayeonSuki** [comment]
Will the batch_size in training stage be the factor influencing the translation quality?
Can I set a larger batch_size like 8 or 16?

**Comment by Zhicaiwww** [comment]
> Will the batch_size in training stage be the factor influencing the translation quality? Can I set a larger batch_size like 8 or 16?

I don’t think this factor matters much. However, a larger batch size with a fixed number of iteration steps may lead to better customization.
